tests/test_leftovers.py

```diff
@@ -0,0 +1,294 @@
+from indexd.blueprint import get_record
+from tests.test_client import get_doc
+import pytest
+from indexd.index.errors import NoRecordFound as IndexNoRecordFound
+from indexd.dos.blueprint import get_dos_record
+import json
+
+
+def test_get_record_call_dist_get_record(client, user):
```

Comment:
i think this test should be renamed to something that makes it easier to understand what is being tested